### PR TITLE
Update INSTALL.txt for PS 8

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update INSTALL.txt for PS 8 and remove some Addons related references
| Type?             | improvement
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | 
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26065)

### CODE REVIEW

**Review by atomiix — DISMISSED** [review]

File: INSTALL.txt [hunk 1]
```diff
@@ -80,8 +80,6 @@ Essential links about PrestaShop:
 * Official blog: https://www.prestashop.com/en/blog
 * Developer blog: https://build.prestashop.com/
 * Get community support: https://www.prestashop.com/forums/
```
Comment (PierreRambaud) [inline INSTALL.txt]:
```suggestion
```

Comment (PierreRambaud) [reply INSTALL.txt]: Same for the user guide above: User Guide: https://doc.prestashop.com/display/PS17/User+Guide

Comment (PierreRambaud) [reply INSTALL.txt]: and the official blog

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
If you start to remove unnecessary links, remove them all :D

File: INSTALL.txt (hunk 1 above)
Comment (matks) [reply INSTALL.txt]: User Guide stays in the project scope I believe.

I dont know if the forum belongs to the project or the company. Slack is for the project, it's a communication channel. Isn't the forum a similar communication channel?

Developer blog also belongs to the project.

My PR does not remove useless items, just addons related items 😉  .

Comment (PierreRambaud) [reply INSTALL.txt]: Everything that is not stored or managed by the open source project has to be removed

Comment (PierreRambaud) [reply INSTALL.txt]: > Everything that is not stored or managed by the open source project has to be removed

They told me to remove all links in the installer interface :sweat_smile:

**Review by PierreRambaud — DISMISSED** [review]

File: INSTALL.txt [hunk 2]
```diff
@@ -40,15 +40,15 @@
...
 
-             --- ===== Installation instructions for PrestaShop 1.7 ===== ---
+             --- ===== Installation instructions for PrestaShop 8 ===== ---
 
 
 
 === Prerequirements
 
-To install PrestaShop 1.7, you need a web server running PHP 7.1.3+ and any flavor of MySQL 5.5+ (MySQL, MariaDB, Percona Server, etc.).
+To install PrestaShop 8, you need a web server running PHP 7.2+ and any flavor of MySQL 5.6+ (MySQL, MariaDB, Percona Server, etc.).
```
Comment (sowbiba) [inline INSTALL.txt]: do we need to put the min patch version : 7.2.5+ ?

Comment (matks) [reply INSTALL.txt]: Yes we can

Comment (matks) [inline INSTALL.txt]:
```suggestion
To install PrestaShop 8, you need a web server running PHP 7.2.5+ and any flavor of MySQL 5.6+ (MySQL, MariaDB, Percona Server, etc.).
```

**Review by sowbiba — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by matks** [comment]
I fixed conflicts using github interface so now github CI blocks my PR because it contains merge commits.

Too much friction, topic is not worth this time. I close.

**Pull request closed without merging**